app/views/locations/index.html.slim

```diff
@@ -1,53 +1,48 @@
 - content_for :title do
   .header-title.text-uppercase Locations
-hr/
-.staff-details.mt20
+div
   .row
-    .col-sm-12
-      = link_to new_location_path, class: 'pull-right' do
-        p.text-center
-          = image_tag 'request-service-icon.png', width: '40%'
-        span.text-center.red.text-uppercase
-          |Add Location
-  .row
-    .col-sm-12
-      .table-responsive
-        table.table.location-table
-          thead
-            th Name
-            th Address1
-            th Address2
-            th Address3
-            th City
-            th State
-            th Zip
-            th Phone Number
-            th Latitude
-            th Longitude
-            th 
-          tbody
-            - @locations.each do |location|
-              tr
-                td
-                  = location.name
-                td
-                  = location.address_1
-                td
-                  = location.address_2
-                td
-                  = location.address_3
-                td
-                  = location.city
-                td
-                  = location.state
-                td
-                  = location.zip
-                td
-                  = location.phone_number
-                td
-                  = location.geography[:latitude]
-                td
-                  = location.geography[:longitude]
-                td
-                  = link_to 'Edit', edit_location_path(location), class: 'btn btn-default'
-                  = link_to 'Destroy', location_path(location), method: :delete, class: 'btn btn-danger ml10'
+    .col-sm-6
+      h3 My Locations
+    .col-md-6.text-center
+      = link_to '#', data: { toggle: 'modal', target: '#addLocation' }, class: 'icon-add pull-right' do
+        i
+          = image_tag 'request-service-icon.png', width: '50'
+        br/
+        span ADD LOCATION
```

Comment:
@deepali1-aconnexion please capitalize text by CSS styles. In html it should be `Add location` or `Add Location`.